test/integration/ethereum/optimisticAuctionRebalanceExtenisonV1.spec.ts

```diff
@@ -0,0 +1,599 @@
+
+      priceAdapter = ConstantPriceAdapter__factory.connect(
+        contractAddresses.setFork.constantPriceAdapter,
+        owner.wallet,
+      );
+      weth = IWETH__factory.connect(contractAddresses.tokens.weth, owner.wallet);
+      collateralAssetAddress = weth.address;
+
+      optimisticOracleV3 = OptimisticOracleV3Interface__factory.connect(
+        contractAddresses.oracles.uma.optimisticOracleV3,
+        owner.wallet,
+      );
+
+      optimisticOracleV3Mock = await deployer.mocks.deployOptimisticOracleV3Mock();
+
+      identifierWhitelist = IIdentifierWhitelist__factory.connect(
+        contractAddresses.oracles.uma.identifierWhitelist,
+        owner.wallet,
+      );
+      const whitelistOwner = await impersonateAccount(await identifierWhitelist.owner());
+      await ethers.provider.send("hardhat_setBalance", [
+        await whitelistOwner.getAddress(),
+        ethers.utils.parseEther("10").toHexString(),
+      ]);
+      identifierWhitelist = identifierWhitelist.connect(whitelistOwner);
+      minimumBond = await optimisticOracleV3.getMinimumBond(collateralAssetAddress);
+
+      integrationRegistry = IntegrationRegistry__factory.connect(
+        contractAddresses.setFork.integrationRegistry,
+        owner.wallet,
+      );
+      const integrationRegistryOwner = await impersonateAccount(await integrationRegistry.owner());
+      integrationRegistry = integrationRegistry.connect(integrationRegistryOwner);
+
+      auctionModule = AuctionRebalanceModuleV1__factory.connect(
+        contractAddresses.setFork.auctionModuleV1,
+        owner.wallet,
+      );
+
+      useAssetAllowlist = false;
+      allowedAssets = [];
+
+      dsEth = SetToken__factory.connect(contractAddresses.tokens.dsEth, owner.wallet);
+
+      baseManager = BaseManagerV2__factory.connect(await dsEth.manager(), owner.wallet);
+      operator = await impersonateAccount(await baseManager.operator());
+      baseManager = baseManager.connect(operator);
+
+      auctionRebalanceExtension = await deployer.extensions.deployOptimisticAuctionRebalanceExtensionV1(
+        baseManager.address,
+        auctionModule.address,
+        useAssetAllowlist,
+        allowedAssets,
+      );
+      auctionRebalanceExtension = auctionRebalanceExtension.connect(operator);
+    });
+
+    addSnapshotBeforeRestoreAfterEach();
+
+    context("when auction rebalance extension is added as extension", () => {
+      beforeEach(async () => {
+        await baseManager.addExtension(auctionRebalanceExtension.address);
+      });
+
+      context("when the product settings have been set", () => {
+        let productSettings: any;
+        let identifier: string;
+        let liveness: BigNumber;
+        beforeEach(async () => {
+          identifier = utils.formatBytes32String("TestIdentifier"); // TODO: Check how do we ensure that our identifier is supported on UMAs whitelist
+          await identifierWhitelist.addSupportedIdentifier(identifier);
```

Comment:
check with UMA

Reply:
The default `ASSERT_TRUTH`  allows settlement to happen in one of two ways:
+ Return the 1 value if the claim is true.
+ Return the 0 value if the claim is false or cannot be resolved.

Here are more details for the identifier:
https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-170.md

When using the `ASSERT_TRUTH` identifier with `assertTruth`, you input the identifier as below: 

`0x4153534552545f54525554480000000000000000000000000000000000000000`